composer.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "shipmonk/doctrine-mysql-optimizer-hints",
+    "description": "Custom SQL walker for Doctrine allowing usage of MySQL optimizer hints without need of native queries",
+    "license": [
+        "MIT"
+    ],
+    "require": {
+        "php": "^7.2 || ^8.0",
+        "doctrine/collections": "^1.6.8 || ^2.0.0",
+        "doctrine/lexer": "^1.1.0 || ^2.0.0",
+        "doctrine/orm": "^2.7.0"
+    },
+    "require-dev": {
+        "doctrine/annotations": "^1.14.3",
+        "editorconfig-checker/editorconfig-checker": "^10.4.0",
+        "ergebnis/composer-normalize": "^2.19.0",
+        "phpstan/phpstan": "^1.10.38",
+        "phpstan/phpstan-phpunit": "^1.3.15",
+        "phpstan/phpstan-strict-rules": "^1.5.1",
```

Comment:
That would require some funky stuff in CI to avoid different PHP versions supported: https://github.com/shipmonk-rnd/doctrine-mysql-optimizer-hints/actions/runs/6575550534/job/17862952898?pr=1